firebase-storage/src/test/java/com/google/firebase/storage/UploadTest.java

```diff
@@ -130,7 +160,7 @@ public void cantUploadToRoot() throws Exception {
         });
 
     // TODO(mrschmidt): Lower the timeout
-    TestUtil.await(task, 300, TimeUnit.SECONDS);
+    TestUtil.await(task, 7, TimeUnit.MINUTES);
```

Comment:
this seems at odds with the line comment above😅 why is this necessary?

Reply:
Ended up changing this to 1 minute. The 7 was a precaution :-)